comp/metadata/inventorychecks/inventorychecksimpl/inventorychecks.go

```diff
@@ -292,3 +297,22 @@ func (ic *inventorychecksImpl) writePayloadAsJSON(w http.ResponseWriter, _ *http
 	}
 	w.Write(scrubbed)
 }
+
+func (ic *inventorychecksImpl) getFilesMetadata() metadata {
+	configFiles := providers.ReadConfigFormats()
+	if len(configFiles) == 0 {
+		ic.log.Errorf("could not read files metadata")
+		return metadata{}
+	}
+
+	filesMetadata := metadata{}
+	for _, configFile := range configFiles {
+		// Use the filename as key
+		filesMetadata[configFile.Filename] = metadata{
+			"raw_config": configFile.ConfigFormat, // convert to string
```

Comment:
nit: please remove this comment as the conversion to string is no longer happening (it's already a string)

```suggestion
			"raw_config": configFile.ConfigFormat,
```